src/Fixer/PhpUnit/PhpUnitTestAnnotationFixer.php

```diff
@@ -305,7 +305,7 @@ private function addTestPrefix($functionName)
             return 'test_'.$functionName;
         }
 
-        return'test'.ucfirst($functionName);
+        return 'test'.ucfirst($functionName);
```

Comment:
Ha, nice - this is the only place where the fixer fixed something so far in the code base!